test/sql/copy/csv/unquoted_escape/basic.test

```diff
@@ -0,0 +1,28 @@
+# name: test/sql/copy/csv/unquoted_escape/basic.test
+# description: Test the parsing of unquoted escape characters
+# group: [unquoted_escape]
+
+statement ok
+PRAGMA enable_verification
+
+statement ok
+CREATE TABLE special_char(a INT, b STRING);
+
+statement ok
+INSERT INTO special_char VALUES
+    (0, E'\\'), (1, E'\t'), (2, E'\n'),
+    (3, E'a\\a'), (4, E'b\tb'), (5, E'c\nc'),
+    (6, E'\\d'), (7, E'\te'), (8, E'\nf'),
+    (9, E'g\\'), (10, E'h\t'), (11, E'i\n'),
+    (12, E'\\j'), (13, E'\tk'), (14, E'\nl'),
+    (15, E'\\\\'), (16, E'\t\t'), (17, E'\n\n'),
+    (18, E'\\\t\n');
+
+loop buffer_size 10 25
+
+query I
```

Comment:
I would prefer to favor simple
`FROM read_csv()` queries when results are fairly small (say less than 100 rows), because they are much easier to debug if things go wrong in the future

Reply:
Same comment for the other test files

Reply:
How can we make sqllogictest handle these special characters properly?

Reply:
Certainly! I understand the importance of making the code future-proof and easy to maintain. I will make all the requested changes shortly.

Reply:
- `basic.test`: Added a test case that uses simple `FROM read_csv` to read `plain.csv` (which does not contain tabs and newlines) and lets sqllogictest compare the results.
- `*.test`: Added comments to explain the CRLF -> LF replacement.